### PR TITLE
Added name to input attribute and translations

### PR DESCRIPTION
**Pull request by coderdiaz**
Thank you for your efforts to make this component.

I've added a property to the text field with the purpose to obtain information within a form, as it did not have the property.

Also, I've added a translation feature.

You can accept the request or not, no problem =).
Best regards.

### CODE REVIEW

**Comment by charliekassel** [comment]
Hey @coderdiaz thank you for your contributions!
I'm currently away for a couple of weeks but will review when I return.

**Comment by coderdiaz** [comment]
@charliekassel It's ok. Greetings.

File: src/utils/DateUtils.js [hunk 1]
```diff
@@ -131,22 +128,22 @@ module.exports = {
...
-    formatDate(date, format) {
+    formatDate(date, format, traslation) {
         let year = date.getFullYear()
         let month = date.getMonth() + 1
         let day = date.getDate()
-        let monthName = this.getMonthName(date.getMonth())
+        let monthName = this.getMonthName(date.getMonth(), traslation.months.original)
         let str = format
+            .replace(/S/, this.getNthSuffix(date.getDate()))
+            .replace(/D/, this.getDayNameAbbr(date, traslation.days))
```
Comment (charliekassel) [inline src/utils/DateUtils.js]: The suffix and day name rules were originally set to run last so as to not be overridden, for instance a suffix added to the third of the month would result in `3rd` and would then get the `d` replaced by the day rule.
Was there another reason you moved these to the top?

File: src/utils/DateUtils.js [hunk 2]
```diff
@@ -131,22 +128,22 @@ module.exports = {
      * @param {Date}
      * @return {String}
      */
-    formatDate(date, format) {
+    formatDate(date, format, traslation) {
```
Comment (charliekassel) [inline src/utils/DateUtils.js]: typo here: tra**n**slation

File: src/utils/DateUtils.js [hunk 3]
```diff
@@ -16,26 +16,24 @@ module.exports = {
      * @param {Date}
      * @return {String}
      */
-    getDayNameAbbr(date) {
+    getDayNameAbbr(date, days) {
```
Comment (charliekassel) [inline src/utils/DateUtils.js]: Can you add the new param to the docblock here please?

File: src/utils/DateUtils.js [hunk 4]
```diff
@@ -16,26 +16,24 @@ module.exports = {
...
     },
 
     /**
      * Return name of the month
      * @param {Number|Date}
      * @return {String}
      */
-    getMonthName(month) {
-        let monthNames = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+    getMonthName(month, months) {
```
Comment (charliekassel) [inline src/utils/DateUtils.js]: Docblock needed here too

File: src/utils/DateUtils.js [hunk 5]
```diff
@@ -131,22 +128,22 @@ module.exports = {
...
      */
-    formatDate(date, format) {
+    formatDate(date, format, traslation) {
         let year = date.getFullYear()
         let month = date.getMonth() + 1
         let day = date.getDate()
-        let monthName = this.getMonthName(date.getMonth())
+        let monthName = this.getMonthName(date.getMonth(), traslation.months.original)
         let str = format
+            .replace(/S/, this.getNthSuffix(date.getDate()))
```
Comment (charliekassel) [inline src/utils/DateUtils.js]: What do you think about adding a translation for the suffix too?

File: package.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "vuejs-datepicker",
-  "version": "0.2.10",
+  "version": "0.2.11",
```
Comment (charliekassel) [inline package.json]: Can you return this to 0.2.10 and once I merge the PR I'll update this with an `npm version patch` command

Comment (coderdiaz) [inline package.json]: Ok, This week I make the appropriate changes. A cordial greeting.

**Comment by charliekassel** [comment]
Great, thanks.
And thank you for your contributions!

File: src/utils/DateUtils.js (hunk 1 above)
Comment (coderdiaz) [inline src/utils/DateUtils.js]: _Yes, the problem is this._
If you pick **September** or **December** in datepicker and the format is _'d MMMM yyyy'_ your return is **'8 theptember 2016'**, the suffix _/S/_ overwrites 'S'eptember to 'Th'eptember.

errordatepicker

It's therefore, I changed the elements initially to overwrites those rules first and then the others.

File: src/utils/DateUtils.js (hunk 5 above)
Comment (coderdiaz) [inline src/utils/DateUtils.js]: I will Analyze how the suffixes will vary in different languages and how it could be adapted. Greetings.

**Comment by coderdiaz** [comment]
The changes are ready!

**Pull request merged**